python/src/spark_rapids_ml/tree.py

```diff
@@ -322,7 +323,7 @@ def _rf_fit(
                 y = pd.concat(y_list)
             else:
                 # should be list of np.ndarrays here
```

Comment:
Is this comment still relevant?  i.e. `_concat_and_free` now returns cupy or numpy array.

Reply:
Will update.